classes/class-p4-master-site.php

```diff
@@ -605,6 +606,27 @@ public function view_meta_box_search( $post ) {
 		<?php
 	}
 
+	/**
+	 * Forces a user to login using Google Auth if they have a greenpeace email
+	 * 
+	 * @param WP_User $user The current user logging in.
+	 * @param String  $username The username of the user.
+	 * @param String  $password The password of the user.
+	 */
+	function enforce_google_signon( $user, $username, $password ) {
+
+
+		if ( defined( 'WP_DEBUG' ) && is_bool( WP_DEBUG ) && WP_DEBUG ) {
+			return $user;
+		}
+
+		if ( strpos( $user->data->user_email, '@greenpeace' ) !== false ) {
+			return new WP_Error( 'login', __( 'You are trying to login with a Greenpeace email. Please use Google login button instead.' ) );
```

Comment:
do we need to add the `planet4-master-theme-backend` text domain here for the translations?